index.html

```diff
@@ -31,7 +31,8 @@ <h3><a href="#" class="action-text">LET’S CONNECT</a></h3>
         <div class="grid-container">
```

Comment:
- [ ] Great job making your works cards look well in a desktop view. However, note that you are supposed to stick as much as possible to the provided Figma design which includes typography, colors, fonts, etc. Kindly make sure that the works cards appear in the alternating form as they are in the Figma design.

**This is what your design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/192340139-78e702d9-f44c-42e9-879e-68d5d623ab17.png)

**This is the desired Figma design:**

![image](https://user-images.githubusercontent.com/86318284/192340140-7a2c5bff-fe7f-4f92-8347-58df1a06f1e6.png)


**Hint:** Give your cards additional classes for those that need to be alternated and use the following one line for the trick! ```flex-direction: row-reverse;```